content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Notation v1.0.0-RC.7 is available!
+author:  "Notation Release Team"
+date: 2023-05-28
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.7, including [Notation CLI v1.0.0-rc.7](https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.7), [notation-go library v1.0.0-rc.6](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.6), and [notation-core-go library v1.0.0-rc.4](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.4). This blog walks you through the updates in this release.
+
+> NOTE: Both Notation CLI v1.0.0-rc.7 and v1.0.0-rc.6 have the same functionalities. However, v1.0.0-rc.7 included an additional fix for an E2E test case.
+
+## What's new
+
+This release adds the following major changes:
+
+- Security advisory fixes
+- Notation commands support reading Docker credentials if the [credentials store](https://docs.docker.com/engine/reference/commandline/login/#configure-the-credentials-store) is not configured
```

Comment:
Notation CLI commands will use [credentials store](https://docs.docker.com/engine/reference/commandline/login/#configure-the-credentials-store)  configured in Docker's config.json when users doesn't specfies it in Notation's config.json

Reply:
I like @priteshbandi  edits better

Reply:
@iamsamirzon @priteshbandi this is already supported by Notation. The new scenario in this release is to read docker credentials if none of credentials stores are present. In previous release, Notation would fail if none of credential stores are present.  Copied below from [docker document](https://docs.docker.com/engine/reference/commandline/login/#credentials-store) the default behavior that when none of credentials stores are present, what docker will do:

>Default behavior
By default, Docker looks for the native binary on each of the platforms, i.e. “osxkeychain” on macOS, “wincred” on windows, and “pass” on Linux. A special case is that on Linux, Docker will fall back to the “secretservice” binary if it cannot find the “pass” binary. If none of these binaries are present, it stores the credentials (i.e. password) in base64 encoding in the config files described above.